packages/plugin-types/src/package_manager_controller.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * TODO: use the latest execa.
+ * execa v8 doesn't support commonjs, so we need to use the types from v5
+ * https://github.com/sindresorhus/execa/issues/489#issuecomment-1109983390
```

Comment:
do we have GH issue for this ?

Reply:
Now we have https://github.com/aws-amplify/amplify-backend/issues/962